src/Paramore.Brighter.Archive.Azure/AzureBlobArchiveProvider.cs

```diff
@@ -0,0 +1,39 @@
+using Azure.Storage.Blobs;
+
+namespace Paramore.Brighter.Storage.Azure;
+
+public class AzureBlobArchiveProvider : IAmAnArchiveProvider
```

Comment:
I think the main difference between this and the luggage store is that with the luggage store we only store the payload, not the whole message, hence our use of a stream

Reply:
Completely agree, it was an optional extra here but I thought it was a Nice Touch however it looks like azure doesn't want to give me access to do this 🤣